lib/tasks/codereview.rake

```diff
@@ -0,0 +1,12 @@
+require 'pronto'
+
+desc 'codereview'
+task :codereview do
+  next unless ENV['PULL_REQUEST_ID'] && ENV['PULL_REQUEST_ID'].downcase != "false"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.